copyright

```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
unintended whitespace change?

Reply:
What did change though?

Reply:
You inserted a byte order mark (BOM) at the beginning of the file.
Please confirm your text editor settings.

Reply:
Fixed, thank you